### DISCUSSION TITLE
2022 GitHub Copilot not working VSCode

### DISCUSSION THREAD

**Discussion by reecenward** [community/community#31061] [Copilot Conversations]
what worked for me:

in visual studio code go to Code > Preferences > Settings

Find Copilot > Then go to Advanced edit in settings.json

Go to top and on the 4th line or so

"editor.inlineSuggest.enabled": false

set to true

"editor.inlineSuggest.enabled": true

**Comment by iamcjwilliams** [comment]
Please refer to this post for assistance with VSCode: https://github.com/orgs/community/discussions/30716
